doc/usecases/transforming_coordinate_systems.adoc

```diff
@@ -0,0 +1,22 @@
+= Transforming coordinate systems
+
+Transforming coordinates for a specific vehicle or sensor is a common task when running simulations.
+OSI uses DIN ISO 8855:2013-11 for coordinate systems and transformations between coordinate systems.
+
+//TODO: Should we add one or more sentences about the mathematical operations involved?
```

Comment:
In my opinion that would be very helpful. Also pictures help here a lot, I think.